src/Compilers/CSharp/Test/Semantic/Semantics/TargetTypedDefaultTests.cs

```diff
@@ -3079,20 +3110,7 @@ static void M(C x)
 ";
 
             var comp = CreateCompilation(source, parseOptions: TestOptions.Regular7_1);
-            comp.VerifyDiagnostics(
-                // (6,13): error CS0019: Operator '==' cannot be applied to operands of type 'C' and 'default'
-                //         _ = x == default
-                Diagnostic(ErrorCode.ERR_BadBinaryOps, "x == default").WithArguments("==", "C", "default").WithLocation(6, 13),
-                // (7,16): error CS0019: Operator '==' cannot be applied to operands of type 'default' and 'C'
-                //             || default == x;
-                Diagnostic(ErrorCode.ERR_BadBinaryOps, "default == x").WithArguments("==", "default", "C").WithLocation(7, 16),
-                // (8,13): error CS0019: Operator '!=' cannot be applied to operands of type 'C' and 'default'
-                //         _ = x != default
-                Diagnostic(ErrorCode.ERR_BadBinaryOps, "x != default").WithArguments("!=", "C", "default").WithLocation(8, 13),
-                // (9,16): error CS0019: Operator '!=' cannot be applied to operands of type 'default' and 'C'
-                //             || default != x;
-                Diagnostic(ErrorCode.ERR_BadBinaryOps, "default != x").WithArguments("!=", "default", "C").WithLocation(9, 16)
-                );
+            comp.VerifyDiagnostics();
```

Comment:
>comp.VerifyDiagnostics(); [](start = 12, length = 25)

For success cases, I think we would want to run the code and observe that they behaves per spec.